thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/lib/FunctionReplayRunnable.java

```diff
@@ -118,22 +127,23 @@ public void run() {
       functionName.append(entry.getValue());
     }
     try {
-      clonedFunctionId = onboardResource.cloneAnomalyFunctionById(tuningFunctionId, functionName.toString(), false);
-      this.lastClonedFunctionId = clonedFunctionId;
+      clonedFunctionId = onboardResource.cloneAnomalyFunctionById(tuningFunctionId, functionName.toString(), cloneAnomaly);
+      this.clonedFunctionId = clonedFunctionId;
```

Comment:
The clonedFunctionId will always be overrided to the new clonedFunctionId, and in the end you only keep the very last cloned function Id. Shall we keep the name of "lastClonedFunctionId" or some what change to a better name to specify this logic?